edgelet/support-bundle/src/support_bundle.rs

```diff
@@ -213,6 +213,10 @@ where
             .runtime
             .list_with_details()
             .collect()
+            // Getting modules requires the management socket, which might not be available if
+            // aziot-edged hasn't started. Require this operation to complete within a timeout
+            // so it doesn't block forever on an unavailable socket.
+            .timeout(std::time::Duration::from_secs(5))
```

Comment:
30s, perhaps? 5s might be too short. Behind the scenes it's one dockerd request for list + multiple dockerd requests for each container's inspect. The inspect requests do run in parallel, but if the system is particularly busy it might end up being slow.

Reply:
Fixed